.github/workflows/merge-release-branch.yml

```diff
@@ -0,0 +1,22 @@
+name: Merge Release Into Master
+
+on: repository_dispatch
```

Comment:
nit feel free to ignore:
We can set` types:` here if we want to restrict the manual trigger even further with a specific `event_type` value in the webhook payload.

```
on:
  repository_dispatch:
    types: [sdk-release-merge]
```

Reply:
Will add this when/if we trigger from a webhook. We'll use the button in the Github UI for now.